packages/to-dts/lib/type.js

```diff
@@ -27,16 +27,14 @@ const primitives = [
   'this',
 ];
 
-function typeFn(g) {
-  const refs = {};
+const ENTRY_RX = /^#\/entries\//;
 
+function typeFn(g) {
   const getBase = (def, tsParent) => {
+    const isEntry = def.path && ENTRY_RX.test(def.path);
```

Comment:
The definitions is deemed to be an API entry when it's path starts with `#/entries`. This results in that all entities in the entries section is considered as API entry.

Note that other types (parameters, return types etc) in the entries section is not considered to be an entry.